### ISSUE TITLE
ME:A stalls and only shows in Task Manager with Mods applied, but can launch ME:A from FFM with no Mods Applied

### ISSUE THREAD

**Issue by IgnusLupin** [GalaxyEham/FrostyToolSuite#92]
GalaxyEham,
     I've been scouring the web, looking for answers but haven't been able to find a solution.  I'm not sure if this is due to FFM or to the mods, but every one I've tried does the same thing.  If I apply a mod, any mod, then launch, FFM will show the "Applying Mods" dialog, then Origin pops up or Flashes on the task bar, and.. nothing.  Task manager shows it active, but no CPU activity.

I've tried:
Removing the ModData folder from the ME Folder
Deleting bcrypt.dll from the ME Folder 
I've taken ownership of all ME:A Files/folders
I've set full control for my users on all files and folders in ME:A
I've ran Origin and FFM as administrator, and not as administrator, as well as one Admin and the other not, and vice/versa.

Not sure what else to do... I've even moved the "Patch" folder from the moddata to the ME:A main folder and merged it with the existing (just to see if it worked) and no go.   And yes, I made a backup of the patch folder before doing so.  After testing, I deleted the patch folder (and moddata, and bcrypt.dll) and renamed the backup patch folder back to "patch" so it's back to normal.

I can run the game without mods and even launch it via FFM with no mods loaded... so I'm kind of at a loss.  I've also tried it with each mod individually, but no go.

I've been using and creating mods for years and I have 20 years of Computer Administration and Support under my belt... so I have a good background when it comes to computers, gaming and modding.  

Not sure if it's FFM 1.0.3.1, ME:A (latest), an issue with Windows 10, or a problem with each one of the 13 mods I got from nexusmods that are causing it not to load but, please help... and thank you in advance.

**Comment by badkarma12** [comment]
How did you fix this?

**Comment by IgnusLupin** [comment]
didn't actually, however I did prove it was permissions... just haven't been able to get help as to how to fix the problem.

I literally tried everything suggested on other sites, even went on Discord but didn't get ANY help from them other than "it's a permissions issue" repeated over and over again.

To get it to work, and as part of my troubleshooting, I activated the local administrator account (mine is an Admin account but it's not the builtin administrator account, for obvious reasons), then removed my system from my domain (moved to a workgroup... disregard this if you don't know what that is, it doesn't affect anyone but power users and those using company laptops).  After which, I disabled my Avast Antivirus and went through the extensive process of making sure all the folders and files involved were A.) on the same drive as the game, B.) took Ownership of them, and C.) had the correct permissions.

After all of this, I tested it and Frosty worked with ME:A... FINALLY!  So, I started to reverse the process one step at a time and test after each step.  (i.e. re-enabled Avast, tested, move the folders to their original locations and verified permissions/ownership hadn't changed, tested, re-joined to my home domain but logged in as local admin, tested, finally switched folder/file ownership and permissions to domain user and logged in as that user, tested-FAILED.

So, my work around is, when I want to play ME:A modded, I log in as the local Admin account and launch it... but as soon as I'm done playing, I log out and back in as my regular user.  And I don't access the Internet under the Local Admin account AT ALL.

Like I said initially, haven't found the fix.  Devs can't explain why it won't work under my other user... not that they discussed it at all... just got "It's a permissions issue."   Which, is technically correct... but not in how they're thinking as everything is owned by the logged in user, which I made sure had full permissions to all files/folders used by this process.  Being an Information Technology Specialist for over a decade, with 20 years of system support, and gaming experience means, at this point, I know how to cover all the bases.  Not saying I catch everything every single time but I know how to make sure permissions are set correctly.  (I've also been a gamer way longer than most my age)

Hope this helps you and others.
